### ISSUE TITLE
Replace point initialization iteration method with direct solver

### ISSUE THREAD

**Issue by jforthofer** [firelab/windninja#247]
We should consider replacing the iterative method with a non-iterating method.  An attempt at doing this would be to just set the Ax=b equations at the points using the "blasting the diagonal" method.  In theory this should work fine numerically, and a converged solution should be reached.  It's not clear of the final wind field would be a reasonable field, because we don't know how the effect of the blasted diagonal (the set velocity value) would propagate out from that location.  Since the solver is also trying to limit the change from the initialized values, it might look ugly, hard to say without trying it.  If it does, there still might be a way to allow the effect of the blasted diagonal to modify the cells around it (in a decaying with distance way) in the Ax=b equation.  Maybe you could "slightly" blast the diagonals of the neighboring cells or something like this, decaying with distance away?

**Comment by nwagenbrenner** [comment]
Related issue #57.

**Comment by jforthofer** [comment]
Thanks Natalie, I forgot about that one...

**Comment by ksshannon** [comment]
Can we get a branch working on this?  Seems like it's worth a try.  /cc @jforthofer

**Comment by jforthofer** [comment]
Yeah, I think this would work and wouldn't be too hard.  If it did work, most of the code modification would involve getting rid of the current iterative loop stuff.  The question is, who would do the work?  I might be able to talk with Mark about this and do it, maybe combine with work adding the new Massman canopy flow model.  Another option might be to have Tanner do it, I could help.

**Comment by nwagenbrenner** [comment]
We talked about this again today. Probably makes sense to take a stab at this now with the station-fetch work. I'll work on this.
